src/components/SearchBar/SearchBar.js

```diff
@@ -15,15 +15,17 @@ class SearchBar extends Component {
 
     const match = text => {
       return this.props.movies.filter(movie => {
-        const textToLower = movie[text].toLowerCase()
-        return textToLower.includes(event.target.value.toLowerCase())
+        return movie[text] ?
```

Comment:
Remember that filter will simply return the movie that has text if the return statement evaluates to true, or nothing if it evaluates to false. It won't return the actual text.